mesh/nurbs.cpp

```diff
@@ -3246,6 +3487,33 @@ void NURBSExtension::SetSolutionVector(Vector &coords, int vdim)
    }
 }
 
+void NURBSExtension::Set1DSolutionVector(Vector &coords, int vdim)
+{
+   Array<const KnotVector *> kv(1);
+   NURBSPatchMap p2g(this);
+
+   weights.SetSize(GetNDof());
+   for (int p = 0; p < GetNP(); p++)
+   {
+      p2g.SetPatchDofMap(p, kv);
+      NURBSPatch &Patch = *patches[p];
```

Comment:
MFEM has (unwritten, unenforced?) style that variable names should begin with a lower-case letter: `Patch` -> `patch`